src/Tasks/Microsoft.NET.Build.Tasks/GenerateBundle.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
+using Microsoft.NET.HostModel.Bundle;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    public class GenerateBundle : TaskBase
+    {
+        [Required]
+        public ITaskItem[] FilesToBundle { get; set; }
+        [Required]
+        public string AppHostName { get; set; }
+        [Required]
+        public bool IncludeSymbols { get; set; }
+        [Required]
+        public string OutputDir { get; set; }
+        [Required]
+        public bool ShowDisgnosticOutput { get; set; }
+
+        protected override void ExecuteCore()
+        {
+            var bundler = new Bundler(AppHostName, OutputDir, IncludeSymbols, ShowDisgnosticOutput);
+            var fileSpec = new List<FileSpec>(FilesToBundle.Length);
+
+            Array.ForEach(FilesToBundle, 
```

Comment:
Why did you change this to Array.ForEach? I liked the foreach loop better.

Reply:
OK @nguerrera, I pushed 25c2b423608690a37da83dbdcecdc55138af67cb which revers back to the foreach() loop.